src/App.tsx

```diff
@@ -1,98 +1,102 @@
-import React, {
-  useEffect, useLayoutEffect, useRef, useState,
-} from "react"
-import Ps from "perfect-scrollbar"
+import React, { useEffect, useLayoutEffect, useRef, useState } from "react";
```

Comment:
can you revert the "semis" changes?

Reply:
Sure, I think it was the change before I noticed the prettierrc problem